evm.md

```diff
@@ -1219,15 +1219,13 @@ The various `CALL*` (and other inter-contract control flow) operations will be d
 ```
 
 -   `#call_____` takes the calling account, the account to execute as, the account whose code should execute, the gas limit, the amount to transfer, and the arguments.
--   `#callWithCode______` takes the calling account, the accout to execute as, the code to execute (as a map), the gas limit, the amount to transfer, and the arguments.
 -   `#return__` is a placeholder for the calling program, specifying where to place the returned data in memory.
 
 ```k
     syntax InternalOp ::= "#checkCall" Int Int
                         | "#call"         Int Int Int Int Int WordStack Bool
-                        | "#callWithCode" Int Int Map WordStack Int Int WordStack Bool
-                        | "#mkCall"       Int Int Map WordStack     Int WordStack Bool
- // ----------------------------------------------------------------------------------
+                        | "#mkCall"       Int Int Int Int WordStack Bool
+ // ------------------------------------------------------------------------
```

Comment:
formatting

Reply:
line up the arguments of `#call` and `#mkCall`, make the horizontal rule the correct length.